rtcbot/base/multiprocess.py

```diff
@@ -150,3 +151,159 @@ def close(self):
             self.__splog.debug("Process did not terminate in time. Killing it.")
             self._producerProcess.terminate()
             self._producerProcess.join()
+
+
+class ProcessSubscriptionConsumer(BaseSubscriptionConsumer):
+    def __init__(
+        self,
+        directPutSubscriptionType=asyncio.Queue,
```

Comment:
Can this be renamed to `defaultSubscriptionType` to be consistent with the producer?

Reply:
Hi!

According to the inherited class `BaseSubscriptionConsumer`, it takes in `directPutSubscriptionType` as an argument.

```python
class BaseSubscriptionConsumer(baseEventHandler):
    """
    A base class upon which consumers of subscriptions can be built.
    The BaseSubscriptionConsumer class handles the logic of switching incoming subscriptions mid-stream and
    all the other annoying stuff.
    """

    def __init__(self, directPutSubscriptionType=asyncio.Queue, logger=None):
        ...
```

Reply:
Oh crap! That was a typo on my part!

Reply:
Sorry for leading you astray there, I haven't had time to look at this code for a while - `directPut` is correct here!